src/engine/framework/BaseCommands.cpp

```diff
@@ -650,10 +653,45 @@ namespace Cmd {
                 const std::string& name = isNamed ? args.Argv(1) : "";
                 const std::string& command = args.EscapedArgs(2 + +isNamed);
                 std::string delay = args.Argv(1 + +isNamed);
-                bool frames = delay.back() == 'f';
 
-                if (frames) {
-                    delay.erase(--delay.end()); //FIXME-gcc-4.6 delay.pop_back()
+                const char front = delay.front();
```

Comment:
This may read outside the array in case of an empty string. Same goes for some other places reading `back()`